src/wp-admin/includes/dashboard.php

```diff
@@ -1555,3 +1401,244 @@ function wp_welcome_panel() {
 </div>
 	<?php
 }
+
+/**
+ * Returns HTML Mark up for the Petitions Dashboard Widget
+ *
+ * @since CP-1.0.0
```

Comment:
Here and elsewhere, please remove the `CP-` prefix and just write `@since 1.0.0`.  We've prefixed WP versions with `WP-` so that it's clear they are from before ClassicPress started, but we don't need to do this for ClassicPress versions.